app/src/main/res/layout/activity_online2.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is there a particular reason why this file is called `activity_online2` ? If not, I think we can safely remove the 2.